cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/AbstractGoal.java

```diff
@@ -85,6 +90,7 @@ public boolean optimize(ClusterModel clusterModel, Set<Goal> optimizedGoals, Opt
       long goalStartTime = System.currentTimeMillis();
       initGoalState(clusterModel, optimizationOptions);
       SortedSet<Broker> brokenBrokers = clusterModel.brokenBrokers();
+      boolean hasNoExcludedBrokersForReplicaMoveWithReplicas = hasNoExcludedBrokersForReplicaMoveWithReplicas(clusterModel, optimizationOptions);
```

Comment:
Nit: What do you think if we rename this variable to be `originallyHasNoExcludedBrokersForReplicaMoveWithReplicas`? IIUC, this name is more accurate since this boolean is used is after rebalancing all brokers, and when this variable is used, readers can be reminded that this variable reflects the `original` state of the cluster model. It makes this logic easier to understand IMO.